geode-management/src/main/java/org/apache/geode/management/operation/RebalanceOperation.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.management.operation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import org.apache.geode.management.api.AsyncOperation;
+import org.apache.geode.management.runtime.RebalanceInfo;
+
+@JsonIgnoreProperties(value = {"uri"}, allowGetters = true)
```

Comment:
found out another way to achieve this without having to annotate each and every implementation: On the `RestfulEndpoint.getUri` method, add an annotation: `@JsonProperty(access = JsonProperty.Access.READ_ONLY)`

Reply:
good find, I have made the change